Not able to install this Repo using Yarn or NPM

We cannot install this repo.  We are getting:

Yarn
```error An unexpected error occurred: "https://registry.yarnpkg.com/react-nosto: Not found".```

NPM
```npm ERR! 404 Not Found - GET https://registry.npmjs.org/react-nosto - Not found
npm ERR! 404 
npm ERR! 404  'react-nosto@*' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)

--- a/README.md
+++ b/README.md
@@ -36,11 +36,11 @@ You should be using React Nosto if you want to:
 
 ##### Yarn:
 
-    yarn add react-nosto
+    yarn add @nosto/nosto-react
 
 ##### NPM:
 
-    npm install --save react-nosto
+    npm install --save @nosto/nosto-react
 
 
 

